src/pages/app/package/[package].vue

```diff
@@ -31,18 +36,59 @@ async function loadAppInfo() {
       .single()
     app.value = dataApp || app.value
 
-    const date_id = new Date().toISOString().slice(0, 7)
-    const { data } = await supabase
-      .from('app_stats')
+    if (cycleStart && cycleEnd) {
+      const { data } = await supabase
+        .from('stats')
+        .select()
+        .eq('app_id', id.value)
+        .eq('action', 'set')
+        .gte('created_at', getConvertedDate2(cycleStart))
+        .lte('created_at', getConvertedDate2(cycleEnd))
+      if (data)
+        updatesNb.value = data.length
```

Comment:
This don't work above 1k, supabase limit the output, you need to use count:
![CleanShot 2023-09-07 at 15 03 28@2x](https://github.com/Cap-go/capgo/assets/4084527/f924de92-6efe-454e-a2d8-fab3adfee2cf)